package.json

```diff
@@ -13,11 +13,11 @@
     "todo-api"
   ],
   "scripts": {
-    "todo-api": "npm start -w todo-api",
-    "angular-todo": "npm start -w angular-todo",
-    "reactjs-todo": "npm start -w reactjs-todo",
-    "start:central-login": "npm start -w central-login",
-    "start:embedded-login": "npm start -w embedded-login",
+    "todo-api": "npm start --workspace todo-api",
```

Comment:
Identical, except used the longhand version to make it clearer to devs/readers.
Took me a while to search for what `-w` does, so figured be explicit.